sklearn/metrics/_ranking.py

```diff
@@ -871,7 +871,10 @@ def precision_recall_curve(y_true, probas_pred, *, pos_label=None, sample_weight
     )
 
     ps = tps + fps
-    precision = np.divide(tps, ps, where=(ps != 0))
+    # Initialize the result array with zeros to make sure that precision[ps == 0]
+    # does not contain uninitialized values. 
```

Comment:
@ogrisel is putting some trailling spaces ;)

```suggestion
    # does not contain uninitialized values.
```

Reply:
Oups ...